test/verify/check-dashboard

```diff
@@ -94,100 +92,100 @@ class TestBasicDashboard(MachineCase, DashBoardHelpers):
         self.wait_dashboard_addresses (b2, [ "localhost" ])
         b.wait_present("#dashboard-hosts a[data-address='localhost'] button.pficon-delete.disabled")
 
-        self.add_machine (b, m2.address)
-        self.wait_dashboard_addresses (b, [ "localhost", m2.address ])
-        self.wait_dashboard_addresses (b2, [ "localhost", m2.address ])
+        self.add_machine (b, "10.111.113.2")
+        self.wait_dashboard_addresses (b, [ "localhost", "10.111.113.2" ])
+        self.wait_dashboard_addresses (b2, [ "localhost", "10.111.113.2" ])
```

Comment:
I still find `m2.address` much more robust, readable, and maintainable (IPv6 or changing networking setup again) than hardcoded and massively duplicated IP numbers. Changing the definition above like `m2 = provision["machine2"]` and using `m2["address"]` perhaps?